### PR TITLE
MUMUP-2047 : Add in responsiveness for widget-card class

### PR DESCRIPTION
**Pull request by timlevett**
+ widget-card had some margin that wasn't cool for mobile. Added in a responsive adjustment.

### Before
screenshot 2015-08-12 at 3 49 19 pm

### After
screenshot 2015-08-12 at 3 48 41 pm

### CODE REVIEW

**Comment by vertein** [comment]
:+1:

File: src/main/webapp/css/buckyless/responsive.less [hunk 1]
```diff
@@ -208,6 +208,11 @@
     padding-top : 20px;
     width : 30px;
   }
+  
+  .widget-body .input-group {
+    margin:10px 2px 30px;
```
Comment (jhanstra) [inline src/main/webapp/css/buckyless/responsive.less]: maybe increase the side margins...5px?

Comment (timlevett) [inline src/main/webapp/css/buckyless/responsive.less]: Can't do anymore than that. It pushes the input over the edge

File: .codenvy/project.json [hunk 2]
```diff
@@ -1 +1 @@
-{"mixinTypes":["contribution"],"builders":{"configs":{},"default":"maven"},"runners":{"configs":{"system:/java/web/jetty9":{"ram":1000,"variables":{},"options":{}}},"default":"system:/java/web/jetty9"},"type":"maven","attributes":{"contribute_branch":["master"],"languageVersion":["1.6"],"contribute_mode":["contribute"],"language":["java"]}}
\ No newline at end of file
+{"mixinTypes":["contribution"],"builders":{"configs":{},"default":"maven"},"runners":{"configs":{"system:/java/web/jetty9":{"ram":1000,"variables":{},"options":{}}},"default":"system:/java/web/jetty9"},"description":"The frame for uw applications","type":"maven","attributes":{"contribute_branch":["upgrade_to_uw_ui_toolkit_v0.3.1"],"languageVersion":["1.6"],"contribute_mode":["contribute"],"language":["java"]}}
```
Comment (timlevett) [inline .codenvy/project.json]: disregard this change, might decide we don't want these project.json files in our projects.

**Comment by apetro** [comment]
:+1:

**Pull request merged**